ta/pkcs11/src/pkcs11_token.c

```diff
@@ -661,16 +669,15 @@ static void close_ck_session(struct pkcs11_session *session)
 {
 	release_active_processing(session);
 
-	/* No need to put object handles, the whole database is destroyed */
+	release_session_find_obj_context(session);
+
+	/* release all session objects */
```

Comment:
remove extra empty line empty + s/release/Release/

Reply:
OK -- I believe it was cleaner with the empty line there -- but removed it.